<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frMetaPerdaCalorica</class>
 <widget class="QMainWindow" name="frMetaPerdaCalorica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Consumo Calórico Diário</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineConsumoCaloricoDiario"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>kcal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Quantidade de Perda de Peso Semanal</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QRadioButton" name="rb1_0">
           <property name="text">
            <string>1,0 kg</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="rb0_3">
           <property name="text">
            <string>0,3 kg</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rb0_5">
           <property name="text">
            <string>0,5 kg</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="rb1_2">
           <property name="text">
            <string>1,2 kg</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="rb1_5">
           <property name="text">
            <string>1,5 kg</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="rb1_4">
           <property name="text">
            <string>1,4 kg</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="rb0_7">
           <property name="text">
            <string>0,7 kg</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="rb0_9">
           <property name="text">
            <string>0,9 kg</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnRetornar">
        <property name="text">
         <string>Retornar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfirmar">
        <property name="text">
         <string>Confirmar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
